apps/vaporgui/VComboBox.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <string>
+#include <QComboBox>
+#include "VContainer.h"
+
+
+//! class VComboBox
+//!
+//! Wraps a QComboBox and provides vaporgui's standard setter/getter functions
+//! and signals.
+
+class VComboBox : public VContainer {
+    Q_OBJECT
+
+public:
+    VComboBox( const std::vector<std::string> &values );
+
+    void SetOptions( const std::vector<std::string> &values );
+    void SetIndex( int index );
+    void SetValue( const std::string& value );
+
+    int GetCurrentIndex() const;
+    std::string GetCurrentString() const;
```

Comment:
The setter SetValue and GetCurrentString refer to the same thing yet they have different names.

Reply:
Are you asking that I implement SetIndex() and SetCurrentString() functions?

SetValue() is somewhat of a standard in the rest of the code base.

Reply:
Im just staying there is an inconsistency in the naming here. Also the Qt5 ` QObject::connect` paradigm does not allow specifying the argument types so we cannot have ValueChanged with two different data types.